lib/yast2_widget_utils.pm

```diff
@@ -44,7 +57,11 @@ sub change_service_configuration {
 }
 
 =head2 change_service_configuration_step
-Modify one service configuration step
+
+ change_service_configuration_step();
+
+Modify one service configuration step.
```

Comment:
Please add a more detailed text that also describes, what the arguments are about.

Reply:
That one as well.